themes/wim/components/wim/_block--menu-top-tasks.scss

```diff
@@ -80,18 +85,17 @@
             right: 10px;
             top: 50%;
             transform: translateY(-50%);
-            content: "\e259";
+            content: "\e259"; //@todo use an @extend for this
```

Comment:
@Crazylord We also discussed this last time. Make use of the classes available in the fontawesome fontpack that we implemented. Extend them like I showed you. We want to be able to only override one class with a different icon font and that it changes site wide. Do not use static values.